src/components/plots/xyPlot.ts

```diff
@@ -156,10 +160,19 @@ module Plottable {
      *   "x" will adjust the x Scale in relation to changes in the y domain.
      *   "y" will adjust the y Scale in relation to changes in the x domain.
      *   "none" means neither Scale will change automatically.
-     *
      * @returns {XYPlot} The calling XYPlot.
      */
-    public autorange(scaleName: string) {
+    public autorange(scaleName: string): XYPlot<X, Y>;
+    public autorange(scaleName?: string): any {
+      if (typeof(scaleName) === "undefined") {
```

Comment:
The convention around the codebase is 
```typescript
if (scaleName == null)
```

We discussed we will change to the bellow soon, but for now, the above is consistent 
```typescript
if (scaleName === void(0))
```